Code/Tools/FBuild/FBuildCore/WorkerPool/JobQueue.cpp

```diff
@@ -587,7 +587,8 @@ void JobQueue::FinishedProcessingJob( Job * job, bool success, bool wasARemoteJo
 			ASSERT( ( result == Node::NODE_RESULT_OK ) || ( result == Node::NODE_RESULT_OK_CACHE ) );
 
 			// (don't check existence of input files)
-			if ( node->GetType() != Node::FILE_NODE )
+			if ( node->GetType() != Node::FILE_NODE &&
+				node->GetType() != Node::EXEC_NODE)
```

Comment:
While reviewing the change for integration into the mainline, I noticed this.
We should always verify the output of a node. This whole change is in order to satisfy this constraint. Without doing this, we could have some very hard to debug build problems.

(The only exception is a FileNode since that is a leaf input to the build which we rely on the dependent node (the compiler) to report (as a missing include typically))

I'll revert this in the mainline integration.

Is this an error or was there some edge case this was covering?